h/models/organization.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+import sqlalchemy as sa
+import slugify
+from xml.etree import ElementTree
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_CHARS = 1
+ORGANIZATION_NAME_MAX_CHARS = 25
+ORGANIZATION_LOGO_MAX_CHARS = 10000
+
+
+class Organization(Base, mixins.Timestamps):
+    __tablename__ = 'organization'
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    name = sa.Column(sa.UnicodeText(), nullable=False, index=True)
+
+    logo = sa.Column(sa.UnicodeText())
+
+    @property
+    def slug(self):
+        """A version of this organization's name suitable for use in a URL."""
+        return slugify.slugify(self.name)
+
+    @sa.orm.validates('name')
+    def validate_name(self, key, name):
+        if not ORGANIZATION_NAME_MIN_CHARS <= len(name) <= ORGANIZATION_NAME_MAX_CHARS:
```

Comment:
I was wondering why this works. TIL that Python's comparisons support [multiple operators](https://docs.python.org/3/reference/expressions.html#comparisons), unlike C, JavaScript etc.

Reply:
I think it would be slightly clearer to write as `if not (... <= ... <= ...)` to make the precedence more obvious.